song-server/src/main/resources/application.yml

```diff
@@ -247,3 +247,13 @@ spring:
 auth:
   jwt:
     public-key-url: http://localhost:8084/oauth/token/public_key
+
+---
+spring.profiles: storageClientCred
```

Comment:
maybe  `score-client-cred` is better???  Kebab case seems better suitted for profile name than camel case